--- ./fexc.c.orig	2013-01-12 20:21:04.000000000 +0100
+++ ./fexc.c	2013-01-13 11:07:59.000000000 +0100
@@ -24,6 +24,7 @@
 #include <sys/mman.h>
 #include <sys/stat.h>
 #include <unistd.h>
+#include <getopt.h>
 #include <fcntl.h>
 
 #define pr_info(...)	errf("fexc: " __VA_ARGS__)
--- ./pio.c.orig	2013-01-12 20:21:04.000000000 +0100
+++ ./pio.c	2013-01-13 11:10:00.000000000 +0100
@@ -25,6 +25,7 @@
 #include <stdint.h>
 #include <string.h>
 #include <unistd.h>
+#include <getopt.h>
 #include <stdlib.h>
 #include <errno.h>
 
